int32[] type #tuple
geometry_msgs/Vector3[] position #list
geometry_msgs/Quaternion[] orientation #list
float64[] discrete_confi #tuple
string[] discrete_type #list
float64[] continuous_confi #tuple
string[] continuous_type #list
